doc/src/section-docs/hashmap.md

```diff
@@ -0,0 +1,33 @@
+Hashmaps as a collection benefit from the same syntactic sugar provided by the
+reader macro with the added support of some generic collection functions, `get`,
+`set!`, and `clear!`.
+
+For retrieving items from a hashmap prefer the reader syntax (<object>.<keyword>) or use the `get`
+function.
+```slosh
+(def m {:one "one", :two "two", "three" 3})
+(assert-equal "one" m.:one)
+(assert-equal "two" m.:two)
+(assert-equal 3 (get m "three"))
+```
```

Comment:
Might mention the ~symbol support here.  For instance '(def one :one) (pro m.~one) should work to allow the use of a variable after the dot (need this for looping with int keys for instance).